tracer/src/Datadog.Trace/Ci/CIEnvironmentValues.cs

```diff
@@ -567,7 +567,12 @@ private void SetupAppveyorEnvironment()
                 Branch = EnvironmentHelpers.GetEnvironmentVariable("APPVEYOR_REPO_BRANCH");
             }
 
-            Message = EnvironmentHelpers.GetEnvironmentVariable("APPVEYOR_REPO_COMMIT_MESSAGE_EXTENDED");
+            Message = EnvironmentHelpers.GetEnvironmentVariable("APPVEYOR_REPO_COMMIT_MESSAGE");
+            string extendedMessage = EnvironmentHelpers.GetEnvironmentVariable("APPVEYOR_REPO_COMMIT_MESSAGE_EXTENDED");
+            if !string.IsNullOrWhiteSpace(extendedMessage)
+            {
+                Message = Message + "\n" + extendedMessage
```

Comment:
Given the "standard" format of log messages, does this need to have a double line break?
```suggestion
                Message = Message + "\n\n" + extendedMessage
```

Reply:
`APPVEYOR_REPO_COMMIT_MESSAGE_EXTENDED` contains everything else after the first line, so it should actually start with a newline in most cases.